src/app/shared/components/metadata-details/metadata-details.component.html

```diff
@@ -0,0 +1,38 @@
+<p *ngIf="status">
```

Comment:
Should be hidden if status is set to None. As it is now, when nothing is chosen for status, this is shown:
![image](https://user-images.githubusercontent.com/3657731/158468742-0726bb32-fbae-4a3d-8711-a8ea8c5f880a.png)